controlers/controlers.js

```diff
@@ -18,3 +22,13 @@ exports.getAllEndpoints = (req, res, next) => {
       next(err);
     });
 };
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  fetchArticlById(article_id)
+    .then((result) => {
+      res.status(200).send(result);
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
Excellent work:

- `getArticleById` is a well named function
- Good work extracting the `article_id` from the params and putting this into a variable, this makes the code cleaner to read
- For the response: get into the habit of sending back an object with a key/value pair. Here your'e just sending the object back but what should be responded with is an object with a key of `article` and then the value would be what comes back from the model, i.e. `.send({ article: result })`. This makes it clearer and the data is being sent back on a specific key.
- On the note of response, have a think about if `result` is a descriptive enough name to represent the data it holds? (it holds a single article...)
- Good work with implementing the catch block to ensure the error handling middleware does its job!